shopify_app.gemspec

```diff
@@ -8,8 +8,9 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = %q{This gem is used to get quickly started with the Shopify API}
 
-  s.add_dependency('rails', '>= 4.2.6')
+  s.required_ruby_version = ">= 2.2.4"
```

Comment:
Yeah `2.2.4` does seem a little arbitrary. @kevinhughes27 what do you think about moving this to either:

`>= 2.2` or `>= 2.2.2` in the next release?